curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efc4f528d6a74d05e68af74.md

```diff
@@ -9,6 +9,13 @@ dashedName: step-56
 
 There's another way to associate an `input` element's text with the element itself. You can nest the text within a `label` element and add a `for` attribute with the same value as the `input` element's `id` attribute.
 
+Here is an example of using the `for` attribute to associate a `label` with an `input` element:
```

Comment:
This one we should definitely keep.
It comes up way to much on the forum
https://forum.freecodecamp.org/search?q=step%2056%20%20%23html-css%20cat%20photo%20app%20after%3A2024-03-01

hopefully the addition of the code example will help with this